app/code/AstoundDRudenko/Badge/Model/Attribute/Badge/Source.php

```diff
@@ -0,0 +1,104 @@
+<?php
+
+declare(strict_types=1);
+namespace AstoundDRudenko\Badge\Model\Attribute\Badge;
+
+use Magento\Catalog\Api\Data\ProductAttributeInterface;
+use Magento\Catalog\Api\ProductAttributeRepositoryInterface;
+use \Magento\Eav\Model\Entity\Attribute\Source\AbstractSource;
+use \Magento\Eav\Model\ResourceModel\Entity\Attribute\Option\CollectionFactory;
```

Comment:
why some classes have \ in front of namespace and some not? please use single pattern for all classes - without a \.